contracts/mainnet/connectors/lido_stETH/helpers.sol

```diff
@@ -0,0 +1,12 @@
+pragma solidity ^0.7.0;
+
+import { TokenInterface } from "../../common/interfaces.sol";
+import { ILido } from "./interface.sol";
+
+abstract contract Helpers {
+	ILido internal constant lidoInterface =
+		ILido(0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84);
+
+	address treasury = 0x28849D2b63fA8D361e5fc15cB8aBB13019884d09; // Instadapp's treasury address
```

Comment:
@bhavik-m if this is not a constant variable, then it during delegate call `treasury` will be 0x0000...0000.
```suggestion
	address internal constant treasury = 0x28849D2b63fA8D361e5fc15cB8aBB13019884d09; // Instadapp's treasury address
```

Reply:
@thrilok209 I will update this in next Pr

Reply:
Why next pr?

Reply:
@thrilok209  SInce we want to make this connector live asap. right? and deploying and enabling the connector will take time.

Reply:
@thrilok209  ohh okay. i was in the misconception that enabling the connector on mainnet takes 2 days. i will redeploy the connector with this change.